packages/gui/src/electron/main.tsx

```diff
@@ -307,8 +328,7 @@ if (!handleSquirrelEvent()) {
           let totalLength = 0;
 
           try {
-            let fileSize: number;
-            fileSize = await getRemoteFileSize(rest.url);
+            const fileSize: number = await getRemoteFileSize(rest.url);
```

Comment:
On line 302, rest.uri should be rest.url:

`          if (allRequests[rest.uri]) {`

should be 
`          if (allRequests[rest.url]) {`